README.md

```diff
@@ -13,3 +13,22 @@ recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+\
+
+///
+Use the in-place method to partition the array. When partitioning, select a base value and divide the array into two parts by exchanging elements: less than or equal to the base value and greater than the base value.\
+
+Then push the index of the unsorted subarray into the stack until the stack is empty.\
+
+If the base value selected each time is the smallest or largest element in the array or a sorted element, each partition will result in an extremely unbalanced partition.\
+
+In the worst case, each partition only reduces n elements and requires n comparisons, with a time complexity of $O(n^2)$\
+
+###\
+https://stackoverflow.com/questions/20917617/whats-the-difference-of-dual-pivot-quick-sort-and-quick-sort\
+
+https://stackoverflow.com/questions/4834740/what-is-the-worst-case-scenario-for-quicksort\
+
+Sources: Provide the principle of pushing the unsorted subarray index into the stack, how to record the pivot position, and help me locate the cause of the error. For the proof part, it comes from the following website\
```

Comment:
That doesn't sound to me like you did anything.

Reply:
Still don't understand what exactly you did. You talk about theorems, but there are no proofs or theorems here. There's no website after "it comes from the following website".